src/components/Header.js

```diff
@@ -2,42 +2,51 @@ import { withAuth0 } from "@auth0/auth0-react";
 import React from "react";
 import AuthButton from "./AuthButton.js";
 import About from "./About.js";
-import { BrowserRouter as Router, Routes, Route, Link } from "react-router-dom";
+// import { BrowserRouter as Router, Routes, Route, Link } from "react-router-dom";
 import Home from "./Home.js";
 import Dashboard from "./Dashboard.js";
 import Container from 'react-bootstrap/Container';
 import Navbar from 'react-bootstrap/Navbar';
 import Nav from 'react-bootstrap/Nav';
 //import Button from 'react-bootstrap/Button';
-import Progress from "./Progress.js";
+// import Progress from "./Progress.js";
+import { Switch, Route, Link } from 'react-router-dom';
 
 
 class Header extends React.Component {
   render() {
     return (
       <>
-        <Navbar style={{margins: 'auto', min: '200vh'}}>
+        <Navbar>
           <Container>
-            <Nav.Item style={{align: 'right'}}>
-              <AuthButton />
-            </Nav.Item>
+            <Navbar.Brand href="#Habitual">Habitual</Navbar.Brand>
           </Container>
-          <Container style={{fontSize: '30px'}}>
-            <Navbar.Brand href="/home">Habitual</Navbar.Brand>
+        </Navbar>
+        <br />
+        <Navbar>
+          <Container>
+
+            <Nav className="me-auto">
+
```

Comment:
Tried to get links to navigate and it is not working.

Reply:
Your `<App>` needs to define which components handle which route (`path). React Router 6 uses [`<Routes>` instead of `<Switch>`](https://github.com/Stratafolio/Stratafolio/issues/1996) like older examples.